<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyNoDialog</class>
 <widget class="QDialog" name="BuyNoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>准备开卡</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>643</width>
     <height>566</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>电话号码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_No">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>绑定小卡：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cbx_mini_card">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>显示套餐：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>套餐类型：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cbx_service">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>资费介绍：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="txt_service_spec">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>187</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>187</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font:12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>开卡面值：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txt_price">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>开通来电显示：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QGroupBox" name="groupBox_ld">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="rdo_ld_on">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>开通</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdo_ld_off">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>不开通</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>开通彩铃：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QGroupBox" name="groupBox_cl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="rdo_cl_on">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>开通</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdo_cl_off">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>不开通</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>客户姓名：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="txi_name">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>证件类型：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="cbx_zjlx">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>证件号码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="txi_indentifyNo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_ld_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="rdo_service_type_on">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>常用</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rdo_service_type_off">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>5</y>
         <width>89</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>所有</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>确认提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>590</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
